skills/nasa/index.js

```diff
@@ -0,0 +1,30 @@
+const request = require('../../api/request');
+
+//put your nasa api key here:
+const key = '';
+
+const intent = () => ({
+    keywords: ['show the picture of the day', "what's today picture", 'show me the picture'],
+    module: 'nasa',
+});
+
+async function nasa_resp() {
+    const nasa_url = 'https://api.nasa.gov/planetary/apod?api_key='+key;
+
+    let data = await request(nasa_url);
+
+    data = JSON.parse(data.body);
+  
+    var title = data.title + ' by ' + data.copyright;
```

Comment:
Avoid using var, it allows weird conflicts with variables of the same name :)

Reply:
Okay, fixed it.